ivy/functional/backends/tensorflow/elementwise.py

```diff
@@ -14,12 +14,15 @@ def abs(
     /,
     *,
     out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+    where: Optional[Union[bool, tf.Tensor, tf.Variable]] = True,
 ) -> Union[tf.Tensor, tf.Variable]:
     if not tf.is_tensor(x):
         x = tf.convert_to_tensor(x)
     x_dtype = ivy.dtype(x)
     if any(("uint" in x_dtype, "bool" in x_dtype)):
         return x
+    if where:
+        return tf.abs(x, where=where)
```

Comment:
`tf.abs` doesn't accept the `where` argument natively, we'll have to use an `ivy.where` composition to implement that behaviour